src/bp-moderation/classes/class-bp-moderation-list-table.php

```diff
@@ -577,11 +577,12 @@ public function column_blocked_member( $item = array() ) {
 		$action_label     = ( 'unsuspend' === $user_action_type ) ? esc_html__( 'Unsuspend', 'buddyboss' ) : esc_html__( 'Suspend', 'buddyboss' );
 
 		// Build actions URL.
-		$view_url = add_query_arg( $moderation_args, bp_get_admin_url( 'admin.php' ) );
+		$view_url           = add_query_arg( $moderation_args, bp_get_admin_url( 'admin.php' ) );
+		$bypass_avatar_url = add_query_arg( array( 'modbypass' => 1 ), bp_core_get_user_domain( $item['item_id'] ) );
```

Comment:
@janakkaneriya007 This is wrong you should use BP_Moderation_Members ->get_permalink function to user link with `modbypass` argument